department-of-veterans-affairs/va.gov-cms
# VACMS-11109: fix for inline guidance widget

## Description

Closes VACMS-11109.

## Testing done
Tested locally and on tugboat.

## Screenshots
![CleanShot 2022-10-05 at 14 16 54](https://user-images.githubusercontent.com/109987005/194165455-13e5aa23-15d2-40ff-822f-ab32fd5d5950.png)


## QA steps
https://pr11113-smcesb5cqx10vsj2stf8liri87lgpbxa.ci.cms.va.gov/

Navigate to Structure -> Content Types -> Full Width Alert -> Manage Form Dispaly.

You should see Paths group. Click on the little gear on the right side, and the form will display. It should now contain the content of the guidance. Make a change, click update, and then click save at the bottom of the form. Click on the gear for Paths Group again, and you should see the change you made. 

### Definition of Done


### Select Team for PR review
